rules/order.js

```diff
@@ -102,35 +176,56 @@ const order = [
     'font',
     'font-family',
     'font-variant',
+    'font-variant-alternates',
+    'font-variant-caps',
+    'font-variant-east-asian',
+    'font-variant-ligatures',
+    'font-variant-numeric',
+    'font-variant-position',
     'font-size',
     'font-size-adjust',
     'font-kerning',
     'font-weight',
     'font-stretch',
     'font-style',
     'font-smoothing',
+    'font-feature-settings',
+    'font-language-override',
+    'font-synthesis',
 
     'text-decoration',
+    'text-decoration-line',
+    'text-decoration-style',
+    'text-decoration-color',
     'text-underline-position',
     'text-transform',
     'text-shadow',
     'text-overflow',
     'text-align',
+    'text-align-last',
     'text-justify',
     'text-indent',
+    'text-emphasis',
+    'text-emphasis-color',
+    'text-emphasis-position',
+    'text-emphasis-style',
+    'text-orientation',
+    'text-rendering',
+    'text-combine-upright',
 
     'line-height',
     'tab-size',
     'white-space',
+    'letter-spacing',
     'word-spacing',
     'word-wrap',
     'overflow-wrap',
+    'hanging-punctuation',
 
     // Paragraphs and wrapping
     'word-break',
+    'line-break',
     'hyphens',
-    'orphans',
```

Comment:
Why was these properties removed?

Reply:
orphans and widows are part of the CSS Columns module, i just moved them near the columns properties.

https://github.com/moxystudio/stylelint-config/pull/39/commits/667620a4d909b3d5650f46dd4c89e865fa589a03#diff-25da83f40ae7af4e718d8b32be574131R102

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Columns

Reply:
👍